packages/agreement-process/Dockerfile

```diff
@@ -34,10 +34,24 @@ RUN pnpm build && \
       packages/agreement-process/dist && \
     find /out -exec touch -h --date=@0 {} \;
 
-FROM node:20.13.1-slim@sha256:cffed8cd39d6a380434e6d08116d188c53e70611175cd5ec7700f93f32a935a6 as final 
+FROM node:20.13.1 as final 
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends wget gnupg ca-certificates && \
+    wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
+    && echo "deb http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list && \
```

Comment:
are we sure there are no licensing issues?
isn't using chromium preferable?

if it can help, you can use [this job](https://github.com/pagopa/interop-be-reports/blob/1.0.x/jobs/ivass-certified-attributes-importer/Dockerfile) as reference